code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -87,6 +88,10 @@ emp_act
 			var/obj/item/clothing/C = bp
 			if(C.body_parts_covered & def_zone.body_part)
 				protection += C.armor[type]
+				
+	if(species)

```

Comment:
@Fox-McCloud You have any suggestions to implement species-based armor protection that /tg/ has? This does not seem like an ideal solution (adding the armor to getarmor_organ).